core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/StackToCloudStackConverter.java

```diff
@@ -202,7 +202,7 @@ InstanceTemplate buildInstanceTemplate(Template template, String name, Long priv
         List<Group> groups = new ArrayList<>();
         Cluster cluster = stack.getCluster();
         if (cluster != null && cluster.getBlueprint() != null) {
```

Comment:
If we are using the BlueprintText, probably we won't need to getBlueprint nullcheck